pkg/microservice/aslan/core/delivery/handler/router.go

```diff
@@ -45,6 +45,14 @@ func (*Router) Inject(router *gin.RouterGroup) {
 		deliveryRelease.GET("/:id", GetDeliveryVersion)
 		deliveryRelease.GET("", ListDeliveryVersion)
 		deliveryRelease.DELETE("/:id", GetProductNameByDelivery, gin2.UpdateOperationLogStatus, DeleteDeliveryVersion)
+
+		deliveryRelease.POST("/helm", CreateHelmDeliveryVersion)
+		deliveryRelease.POST("/helm/global-variables", ApplyDeliveryGlobalVariables)
+		deliveryRelease.GET("/helm/charts", DownloadDeliveryChart)
+		deliveryRelease.GET("/helm/charts/version", GetChartVersionFromRepo)
+		deliveryRelease.GET("/helm/charts/preview", PreviewGetDeliveryChart)
+		deliveryRelease.GET("/helm/charts/filePath", GetDeliveryChartFilePath)
+		deliveryRelease.GET("/helm/charts/fileContent", GetDeliveryChartFileContent)
```

Comment:
move to another group called helm

Reply:
api url path will be adjusted in next feature develop pr